2024/wg-fedid.html

```diff
@@ -167,17 +167,26 @@ <h2>Motivation and Background</h2>
+        The Federated Identity Working Group defines Web Platform features that allow user agents to support secure and privacy-preserving interactions related to digital identities. 
       </p>  
+      <p>
+        These features are intended to support different interaction flows (e.g., authentication, authorization, requesting identities or credentials, and issuance) in a 'traditional' federated identity model - with Identity Providers (IdPs) or Relying Parties (RPs) - and in a digital wallet 'decentralized' model - with Issuers, Holders, and Verifiers.
       <p>
         If any mechanisms developed to support authentication and authorization flows would cause breaking changes for existing protocols, work on that mechanism must include a well-documented transition period.
       </p>
-
-      <section id="section-out-of-scope">
-        <h3 id="out-of-scope">Out of Scope</h3>
-        <p>
-          The identity space is much larger than that of federated authentication and digital credential wallets. While several topics related to identity may be of interest, they are out of the scope for our work.
-        </p>
+      <p>
+        Here, 'privacy' minimally refers to the appropriate processing of personal information and preventing third parties from learning anything about the end-user's environment (e.g., which wallets are available, their brand, and their capabilities). 
+      </p>
+      <p>
+        This group develops new mechanisms that define how information is passed by the user agent between the different entities to facilitate federated and digital identities:
+      </p>
+      <ul>
+        <li>Enabling a Federated Identity Model while adhering to <a href="https://www.w3.org/TR/privacy-principles/" rel="nofollow">privacy principles</a> despite the <a href="https://www.w3.org/2001/tag/doc/web-without-3p-cookies/" rel="nofollow">deprecation of third-party cookies</a>, a cornerstone of such operations.<li>
+        <li>Enabling a Decentralized Identity Model by invoking a wallet without <a href="https://github.com/WICG/digital-identities/blob/main/custom-schemes.md" rel="nofollow">custom schemes</a>, since they have <a href="https://github.com/WICG/digital-identities/blob/main/custom-schemes.md#can-wallets-reliably-determine-their-invoker" rel="nofollow">security</a>, <a href="https://github.com/WICG/digital-identities/blob/main/custom-schemes.md#what-are-the-privacy-implications-of-a-wallet-accepting-custom-schemes" rel="nofollow">privacy</a>, and <a href="https://github.com/WICG/digital-identities/blob/main/custom-schemes.md#user-experience-concerns" rel="nofollow">user experience implications</a> and cannot work reliably or securely in cross-device scenarios.</li>
```

Comment:
Nit: for custom-schemes in particular, it's not that they're insecure in cross-device - there just isn't anything for the browser to invoke (eg. when running on a desktop with the wallet on a phone), and so the developer has to fall back to something less secure like a QR code. Maybe just simplify like this?

```suggestion
        <li>Enabling a Decentralized Identity Model by invoking a wallet without <a href="https://github.com/WICG/digital-identities/blob/main/custom-schemes.md" rel="nofollow">custom schemes</a>, since they have <a href="https://github.com/WICG/digital-identities/blob/main/custom-schemes.md#can-wallets-reliably-determine-their-invoker" rel="nofollow">security</a>, <a href="https://github.com/WICG/digital-identities/blob/main/custom-schemes.md#what-are-the-privacy-implications-of-a-wallet-accepting-custom-schemes" rel="nofollow">privacy</a>, and <a href="https://github.com/WICG/digital-identities/blob/main/custom-schemes.md#user-experience-concerns" rel="nofollow">user experience implications</a> and cannot work in cross-device scenarios.</li>
```